### PR TITLE
Test python services

### PR DESCRIPTION
**Pull request by mikaelarguedas**
connects to https://github.com/ros2/rosidl/issues/67

### CODE REVIEW

**Comment by mikaelarguedas** [comment]
cross rcl service tests are passing on linux :tada:  [Build Status](http://ci.ros2.org/job/ci_linux/1912)

CI for all platforms with linter warnings fixed:
Linux: [Build Status](http://ci.ros2.org/job/ci_linux/1913)
OSX: [Build Status](http://ci.ros2.org/job/ci_osx/1471/)
Windows: [Build Status](http://ci.ros2.org/job/ci_windows/1866/)

**Comment by mikaelarguedas** [comment]
Looks like cross rcl services tests fail on windows. Should we delay merging this until a `wait_for_service` function is implemented in rclpy ? Or merge this and explicitely disable those on windows for now ?

**Comment by dirk-thomas** [comment]
We could raise `SkipTest` for these tests on Windows for now.

**Comment by mikaelarguedas** [comment]
Done in https://github.com/ros2/system_tests/pull/175/commits/f344eb194ab794c1a8c4badd344abb784970d195, now all blacklisted `test_communication` tests will be skipped and will be displayed as such on jenkins: e.g. http://ci.ros2.org/job/ci_windows/1894/testReport/
Ready for review now

File: test_communication/CMakeLists.txt [hunk 1]
```diff
@@ -66,20 +66,26 @@ if(BUILD_TESTING)
...
-            ENV RCL_ASSERT_RMW_ID_MATCHES=${rmw_implementation})
-          set_tests_properties(
-            "${target}${target_suffix}__${TEST_MESSAGE_TYPE}"
-            PROPERTIES REQUIRED_FILES "$<TARGET_FILE:${target}${target_suffix}>"
-          )
+        if(
+          (TEST_MESSAGE_TYPE STREQUAL "BoundedArrayPrimitives" OR TEST_MESSAGE_TYPE STREQUAL "DynamicArrayPrimitives") AND
+          rmw_implementation1 STREQUAL "rmw_opensplice_cpp"
+        )
+          set(SKIP_TEST "True")
```
Comment (mikaelarguedas) [inline test_communication/CMakeLists.txt]: It is working for nose test only indeed, thanks for catching it. Fixed in https://github.com/ros2/system_tests/pull/175/commits/8938b1f1a89c7156aadcd0a0a31eae8b200d70ca
Current CI doesn't test opensplice so it wouldn't have appeared in the CI test results

**Pull request merged**